src/style.css

```
@@ -0,0 +1,51 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  box-sizing: inherit;
+}
+
+div.stars {
+  display: flex;
+  flex-direction: row;
+  box-sizing: inherit;
+}
+
+div.stars .stars__star {
+  box-sizing: inherit;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  padding: 1px 1.5px 3px;
+}
```

```suggestion
.stars__star {
  box-sizing: inherit;
  background-image: url("images/star.svg");
  background-repeat: no-repeat;
  background-position: center;
  width: 16px;
  height: 16px;
  margin-right: 4px;
}
```

використовуй цілі числа